sql/sql_connect.cc

```diff
@@ -1000,6 +1000,28 @@ static int check_connection(THD *thd)
     bzero((char*) &net->vio->remote, sizeof(net->vio->remote));
   }
   vio_keepalive(net->vio, TRUE);
+
+  if (global_system_variables.keepalive_probes != 9)
```

Comment:
@vaintroub ok, this part is done. Please have a look:
* Structure - https://github.com/leoleovich/server/blob/10.2/include/violite.h#L54-L59
* Universal "set" function definition - https://github.com/leoleovich/server/blob/10.2/vio/viosocket.c#L533-L569
* Universal "get" current socket options - https://github.com/leoleovich/server/blob/10.2/vio/viosocket.c#L571-L607
* Universal "set" function usage -
 https://github.com/leoleovich/server/blob/10.2/sql/sql_connect.cc#L1005-L1008
* Default - https://github.com/leoleovich/server/blob/10.2/sql/sql_const.h#L115-L121